test/onigumo_cli_test.exs

```diff
@@ -60,6 +60,12 @@ defmodule OnigumoCLITest do
       end
     end
 
+    for switch <- ["-h", "--help", "--help -C prdel", "-h --invalid-switch"] do
```

Comment:
I believe that there are two separate user stories:

* The help message is explicitly requested by providing the `-h`/`--help` switch.
* Invalid arguments are provided and the help message gets printed as a side effect.

I’d suggest then splitting this test into separate ones for each case.

```suggestion
    for switch <- ["-h", "--help"] do
    end
    …
    for switch <- ["--help -C prdel", "-h --invalid-switch"] do
    end
```